### ISSUE TITLE
Remove deprecated/unused files for compiling DOSBox-X for platform X, Y or Z.

### ISSUE THREAD

**Issue by aybe** [joncampbell123/dosbox-x#1898]
**Describe the bug**

Following #1863, #1191, some bits are misleading for compiling the project, namely:

- CMakeLists.txt (not maintained)
- vs2015/dosbox-x.vcproj (old format, now is .vcxproj instead)

**Describe the solution you'd like**

- remove unused/obsolete/misleading files to make things clear.
- update relevant documentation

**Comment by Jookia** [comment]
I've opened #1900 and #1899 for this.

**Comment by aybe** [comment]
Okay, you've been faster than me, I was going to push the same!

🥇

**Comment by aybe** [comment]
@aybe when #1899 is merged, close this

**Comment by Jookia** [comment]
I've been over the past day or so trying to set up a build environment for Windows. There's currently this many build systems:

- vs2013 (broken)
- vs2015
- nmake makefiles
- mingw makefiles
- autotools

It would really be nice to drop the makefiles and the vs2013 project eventually as they seem to be prone to bitrot. The only person using these seems to be @yksoft1 and they're not being maintained.

**Comment by fr500** [comment]
What are you gonna be using in the future?
please week mingw buildfiles supported

**Comment by Jookia** [comment]
What do you mean mingw buildfiles? Right now I think autoconf is used
for builds?

**Comment by fr500** [comment]
Makefiles, sorry.
I use MSYS2/MingW and I build dosbox, and X with autotools/make

**Comment by Jookia** [comment]
Are you using the makefiles in ref-static-makfiles? This wouldn't affect autotools or the ./build-mingw-* scripts

**Comment by aybe** [comment]
To sum it up, vs2013 and nmake makefiles can be removed, correct ?

**Comment by joncampbell123** [comment]
@yksoft1 I will approve of the removal of those files if you do not respond by Saturday October 10th.

**Comment by Jookia** [comment]
Yes, I think vs2013, nmake and mingw makefiles can be removed safely.

**Comment by yksoft1** [comment]
> I've been over the past day or so trying to set up a build environment for Windows. There's currently this many build systems:
> 
> * vs2013 (broken)
> * vs2015
> * nmake makefiles
> * mingw makefiles
> * autotools
> 
> It would really be nice to drop the makefiles and the vs2013 project eventually as they seem to be prone to bitrot. The only person using these seems to be @yksoft1 and they're not being maintained.

As long as the nmake makefiles are fixed now, VS2013 solution is not needed at least for the master branch anymore.

**Comment by Jookia** [comment]
Can the mingw makefiles be ditched?
